### ISSUE TITLE
button bridge

### ISSUE THREAD

**Issue by hannoverRed** [rdmtc/RedMatic-HomeKit#336]
hallo zusammen, 

muss man für jeden neuen button eine neu bridge anlegen mit separatem port?
lg

**Comment by ptweety** [comment]
Nein, über eine Bridge kannst du recht viele Geräte exponieren. (ich glaube, es waren so ~ 100)

**Comment by hobbyquaker** [comment]
Nein – eine Bridge trägt bis zu 149 Accessories (mehr lässt HAP-NodeJS nicht zu; 4.0.0 meldet das Limit im Editor). Erst darüber braucht es eine zweite Bridge mit eigenem Port.

---
*Written by Claude Fable on behalf of hobbyquaker.*
